docs/release_notes/pending_release.rst

```diff
@@ -1,26 +0,0 @@
-SMQTK Pending Release Notes
```

Comment:
On a separate commit, can you add an empty version of this file back in, in preparation for future changes? Like the following
```
Pending Release Notes
=====================

Updates / New Features
----------------------

Fixes
-----
```